assets/queries/openAPI/missing_required_property/query.rego

```diff
@@ -0,0 +1,45 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	objects := {"paths", "info"}
+
+	object.get(doc, objects[x], "undefined") == "undefined"
```

Comment:
I think, here, we could use `check_field` too, no?

Reply:
The function `check_field` was implemented to resolve cases when the YAML file has 'info' empty. (YAML parser puts info set to null when it is empty (when set to null, `object.get(field, value, "undefined") == "undefined"` does not return a true positive result, because it can not access the 'field'). In the case of checking doc, I think it is unnecessary to use `check_field` since 'doc' is never null (it has 'id' and 'file' set, for example)